tests/error-all/expected_results.json

```
@@ -0,0 +1,24 @@
+{
+  "status": "fail",
```

Seeing how all the tests have the `error` status, this should also have the `error` status:


```suggestion
  "status": "error",
```

---

Interesting. I understand now and will change this, but I think the docs should be updated.

https://exercism.org/docs/building/tooling/test-runners/interface#h-status

It says "tests were run" but I think it means "tests were completed", because I thought "the tests ran but they errored", but of course you could say they didn't run because they errored.

---

If you have a better phrasing, let us know!

---

I am suggesting "The error status should only be used if all of the tests errored."

I also find the example for interpreted languages a little distracting. Are you saying that syntax errors that prevent a file from parsing should be treated differently to other kinds of runtime errors, or are you just using that an example of a runtime error?

I suggest:
"The error status should only be used if all of the tests errored. For compiled languages this is generally a result of the code not being able to compile. For interpreted languages this is a runtime error, such as a syntax error that stops the file from parsing."